### ISSUE TITLE
dcos cli

### ISSUE THREAD

**Issue by thbeh** [mediative/mesos-stack#38]
Some of the dcos cli commands seems deprecated as the version installed is 2.0. e.g.
I could not get to install cassandra because  I couldn't run

dcos package repo add Universe https://universe.mesosphere.com/repo

Any clues?

**Comment by jonas** [comment]
We are currently on the old version of the cli as well as the old repo version. A lot was changed before DC/OS was open sourced. Also note that we are currently considering to switch completely to DC/OS.

```
> dcos --version
dcos version 0.3.6
> grep version- ~/.dcos/dcos.toml
sources = [ "https://github.com/mesosphere/universe/archive/version-1.x.zip",]
```

**Comment by thbeh** [comment]
Yup, i did the downgrade to 0.36 but somehow neither cassandra or spark could run. Yet to dig in deeper on the logs to see why but suspect docker issue...

**Comment by thbeh** [comment]
BTW, I am running on 4 bare metals, using the same ansible roles.

**Comment by thbeh** [comment]
Any recommendations on getting cassandra to work on the cluster?

**Comment by dvictor1** [comment]
Have you provided principal/secret pair, installing a framework?

**Comment by thbeh** [comment]
I saw it in mesos-hdfs but how to you do that for  Cassandra?

**Comment by thbeh** [comment]
I assume you are referring to deploying cassandra using dcos-cli

**dcos install package cassandra**
